MdeModulePkg/Bus/Pci/PciBusDxe/PciOptionRomSupport.c

```diff
@@ -641,12 +641,12 @@ RomDecode (
 /**
   Load and start the Option Rom image.
 

```

Comment:
Please fix the spelling in the commit title, and please don't put PR references in there. If you need to refer to another patch, use the commit hash.